v4/source/teams.yaml

```diff
@@ -657,13 +657,19 @@
           description: Team GUID
           required: true
           type: string
+        - name: graceful
+          in: query
+          description: 'Instead of aborting the operation if a user cannot be added, return an object in form of `{"added_members": [], "errors":{"user_id":"error"}}`'
+          required: false
+          type: boolean
         - in: body
           name: body
           required: true
           schema:
             type: array
             items:
-              $ref: '#/definitions/TeamMember'
+                $ref: '#/definitions/TeamMember'
```

Comment:
Whitespace looks wrong here. Is this change intended? I'm never sure what this kind of syntax gets upset about with indentataion.